support-frontend/.eslintrc

```diff
@@ -57,6 +57,7 @@
       }
     ],
     "react/default-props-match-prop-types": [2, { "allowRequiredDefaults": true }],
+    "react/jsx-closing-tag-location": "off"
```

Comment:
it was messing with the indentation, then complaining about the indentation

Reply:
I think the way to get what we want without disabling this rule would be to do:

```tsx
You can amend your email preferences at any time via{' '}
<a href="https://manage.theguardian.com">
  your account
</a>
```

You have to do the unfortunate `{' '}` to add the space when adding the `a` tag on the next line. I think enforcing the alignment of the tags is probably something we want, but ultimately we should really just add `prettier` and let it deal with all this. That's a future PR so I'm happy to change or leave this for now

Reply:
I see, thanks, I would never have worked this out!

Reply:
np!